com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -373,7 +373,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed the earth curvature not being properly taken into account when evaluating the sun attenuation (case 1357927).
 - Reduced the volumetric clouds pattern repetition frequency (case 1358717).
 - Fixed the clouds missing in the ambient probe and in the static and dynamic sky.
-- Fixed lens flare not rendering correctly with TAAU or DLSS
+- Fixed lens flare not rendering correctly with TAAU or DLSS.
+- Fixed assert failure when enabling the probe volume system for the first time.
+- Significantly improved performance of APV probe debug.
```

Comment:
note for later: APV isn't an official feature (yet) so we don't need to put changelog.